pkg/reconciler/eventtypes.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconciler
+
+import (
+	"context"
+	"fmt"
+
+	"go.uber.org/zap"
+	"k8s.io/apimachinery/pkg/api/equality"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	eventingv1alpha1 "knative.dev/eventing/pkg/apis/eventing/v1alpha1"
+	"knative.dev/pkg/logging"
+
+	"knative.dev/sample-source/pkg/apis/samples/v1alpha1"
```

Comment:
Same comment about aliasing this to `samplesv1alpha1`

Reply:
we don't do this normally for repos like this.

Reply:
Which is alright for this minimal example, but aliasing APIs is a common Kubernetes practice no matter the context. Besides it helps identifying the groups of objects in the code and is a necessary evil when you start having multiple API groups